_site/create/downloads.html

```diff
@@ -241,15 +241,15 @@ <h1 id="raspberry-pi-image">Raspberry Pi image</h1>
 
 <p>We have also created a Linux distribution for the Raspberry Pi, based on the official Raspbian Jessie Lite, but that automatically runs the Prynth software out-of-the-box.</p>
 
-<p>Download the image file and head to the documentation for <a href="../documentation/install-rpi-image.html">installation details</a>.</p>
+<p>Download the image file and head to the documentation for <a href="documentation/install-rpi-image.html">installation details</a>.</p>
 
 <p><a href="https://www.dropbox.com/s/4oydarjeqh2m27f/2016-11-10-prynth.img.zip?dl=1" class="btn btn--warning">Download Prynth RPi image (2016-11-10)</a></p>
 
 <hr />
 
 <h1 id="teensy-source-code">Teensy source code</h1>
 
-<p>Download the Teensy source file program. In the documentation you’ll find more instructions on installing the Teensyduino library and <a href="../documentation/teensy-upload.html">uploading the firmware</a> to the Teensy.</p>
+<p>Download the Teensy source file program. In the documentation you’ll find more instructions on installing the Teensyduino library and <a href="documentation/teensy-upload.html">uploading the firmware</a> to the Teensy.</p>
```

Comment:
Should be documentation/teensy-firmware.html I've got no clue how to fix that in this web interface..